File: starter-bots/NETCoreBot/NETCoreBot/Models/GameObject.cs
using NETCoreBot.Enums;
using System;
using System.Collections.Generic;

namespace NETCoreBot.Models
{
    public class GameObject
    {
        public Guid Id { get; set; }
        public GameObjectType GameObjectType { get; set; }
        public Position Position { get; set; }

        public GameObject(GameObjectType type)
        {
            Id = new Guid();
            GameObjectType = type;
        }

    }
}

File: starter-bots/JavaBot/src/main/java/models/GameObject.java
package models;

import java.util.UUID;

public abstract class GameObject {
    private UUID id;
    private int gameObjectType;
    private Position position;

    public GameObject(int gameObjectType) {
        this.gameObjectType = gameObjectType;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public int getGameObjectType() {
        return gameObjectType;
    }

    public void setGameObjectType(int gameObjectType) {
        this.gameObjectType = gameObjectType;
    }

    public Position getPosition() {
        return position;
    }

    public void setPosition(Position position) {
        this.position = position;
    }

}
